packages/entity/src/ComposedEntityCacheAdapter.ts

```diff
@@ -26,9 +27,13 @@ export default class ComposedEntityCacheAdapter<TFields> extends EntityCacheAdap
     fieldValues: readonly NonNullable<TFields[N]>[]
   ): Promise<ReadonlyMap<NonNullable<TFields[N]>, CacheLoadResult<TFields>>> {
     const retMap = new Map<NonNullable<TFields[N]>, CacheLoadResult<TFields>>();
+    const fulfilledFieldValuesByCacheIndex: NonNullable<TFields[N]>[][] = this.cacheAdapters.map(
+      () => []
+    );
```

Comment:
optional: I've started seeing this syntax around for creating an array of specific length with initialized values. not sure what the relative performance is.

```suggestion
    const fulfilledFieldValuesByCacheIndex: NonNullable<TFields[N]>[][] = Array.from(
      { length: this.cacheAdapters.length },
      () => []
    );
```